NitroxClient/GameLogic/Vehicles.cs

```diff
@@ -212,10 +214,14 @@ public void UpdateVehiclePosition(VehicleMovementData vehicleModel, Optional<Rem
                 if (subRoot)
                 {
                     mvc = subRoot.gameObject.EnsureComponent<MultiplayerCyclops>();
-                    subRoot.GetComponent<LiveMixin>().health = vehicleModel.Health;
                 }
                 else if (vehicle)
                 {
+                    if (vehicle.docked)
+                    {
+                        Log.Debug($"For vehicle {vehicleModel.Id} position update while docked, will not execute");
```

Comment:
I think this might be very noisy... maybe combine the two conditons into just the else if.